tests/2017-12-01/Microsoft.Compute.tests.json

```diff
@@ -0,0 +1,1329 @@
+{
+  "tests": [
+    {
+      "name": "VirtualMachines - Basic",
+      "definition": "https://schema.management.azure.com/schemas/2017-12-01/Microsoft.Compute.json#/resourceDefinitions/virtualMachines",
+      "json": {
+        "apiVersion": "2017-12-01",
+        "type": "Microsoft.Compute/virtualMachines",
+        "name": "[concat(variables('vmName'), copyindex())]",
+        "location": "[variables('location')]",
+        "tags": {
+          "Tag1": "ManagedVM"
+        },
+        "copy": {
+          "name": "VMLoop",
+          "count": "[parameters('numberOfVMInstances')]"
+        },
+        "dependsOn": [
+          "[concat('Microsoft.Network/networkInterfaces/', variables('nicName'), copyindex())]"
+        ],
+        "properties": {
+          "availabilitySet": {
+            "id": "[resourceId('Microsoft.Compute/availabilitySets', 'availabilitySet1')]"
+          },
+          "hardwareProfile": {
+            "vmSize": "[variables('vmSize')]"
+          },
+          "osProfile": {
+            "computerName": "[concat(variables('vmName'), copyindex())]",
+            "adminUsername": "[parameters('adminUsername')]",
+            "adminPassword": "[parameters('adminPassword')]"
+          },
+          "storageProfile": {
+            "imageReference": {
+              "publisher": "[variables('imagePublisher')]",
+              "offer": "[variables('imageOffer')]",
+              "sku": "[parameters('windowsOSVersion')]",
+              "version": "latest"
+            },
+            "dataDisks": [
+              {
+                "lun": 0,
+                "createOption": "Empty",
+                "diskSizeGB": 1023
+              }
+            ]
+          },
+          "networkProfile": {
+            "networkInterfaces": [
+              {
+                "id": "[resourceId('Microsoft.Network/networkInterfaces', concat(variables('nicName'), copyindex()))]"
+              }
+            ]
+          }
+        }
+      }
+    },
+    {
+      "name": "AvailabilitySet",
+      "definition": "https://schema.management.azure.com/schemas/2017-12-01/Microsoft.Compute.json#/resourceDefinitions/availabilitySets",
+      "json": {
+        "type": "Microsoft.Compute/availabilitySets",
+        "name": "availabilitySet1",
+        "apiVersion": "2017-12-01",
+        "location": "[variables('location')]",
+        "properties": {
+          "platformFaultDomainCount": 2,
+          "platformUpdateDomainCount": 2,
+          "managed": true
+        }
+      }
+    },
+    {
+      "name": "Managed Data Disk",
+      "definition": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.Compute.json#/resourceDefinitions/disks",
```

Comment:
>2018-04-01 [](start = 65, length = 10)

can we move the tests for 2018-04-01 to a separate folder?

Reply:
Sure